.github/workflows/test-docker.yml

```diff
@@ -42,3 +42,26 @@ jobs:
         # The Pillow user in the docker container is UID 1000
         sudo chown -R 1000 $GITHUB_WORKSPACE
         docker run -v $GITHUB_WORKSPACE:/Pillow pythonpillow/${{ matrix.docker }}:${{ matrix.dockerTag }}
+        sudo chown -R runner $GITHUB_WORKSPACE
+
+    - name: After success
+      if: success()
+      run: |
+        pip install wheel
+        sudo apt-get install -qq ruby-dev
+        PATH="$PATH:~/.local/bin"
+        .travis/after_success.sh
+      env:
+        MATRIX_OS: ${{ matrix.os }}
```

Comment:
```suggestion
```

There's no `matrix.os` here, and it's used in `after_success.sh` like this, so we can remove it from here.

```sh
if [[ "$MATRIX_OS" == "macOS-latest" ]]; then
    brew install lcov
else
    sudo apt-get -qq install lcov
fi
```